### DISCUSSION TITLE
I want transcription in English Script instead of Hindi for Hindi audio

### DISCUSSION THREAD

**Discussion by HardikJain02** [openai/whisper#725] [Q&A]
I am using [yt_whisper](https://github.com/m1guelpf/yt-whisper) so that I can directly transcribe the video with vtt file by a youtube link. (No need to download the video) Please let me know how can I achieve it.

I am getting the following output

![image](https://user-images.githubusercontent.com/69311953/208459876-1957b3af-7251-414b-925d-62396251ce49.png)

And, I want output like

7) Sir aap toh jaante hai

10) Pichle kuch dino se meri mother ki bahut tabiyat kharab hai

13) Mein eklauta ladka hoon

**Comment by fitbyprabhat** [comment]
Did you got any answer?

**Comment by phineas-pta** [comment]
you have to use another tool to convert from devanagari to latin script

in python you can try https://github.com/virtualvinodh/aksharamukha-python
